django/forms/widgets.py

```diff
@@ -146,8 +146,14 @@ def merge(*lists):
 
     def __add__(self, other):
         combined = Media()
-        combined._css_lists = self._css_lists + other._css_lists
-        combined._js_lists = self._js_lists + other._js_lists
+        combined._css_lists = self._css_lists[:]
+        combined._js_lists = self._js_lists[:]
+        for item in other._css_lists:
+            if item not in self._css_lists:
```

Comment:
I think you could skip this check if `item` is falsy. I haven't done any measurements but I'd suggest changing this to:

`if item and item not in self._css_lists`

Reply:
Including falsy is a user's mistake and we don't remove them from the source lists, so I think we shouldn't change this.

Reply:
@felixxm I was thinking about only specifying `css` OR `js` – the default values are empty datastructures which are falsy as well. There's no point in iterating through `_js_lists` or `_css_lists` to find whether an empty datastructure already exists in there. But it's probably a pointless microoptimization.

Reply:
Agreed. I updated PR and added tests.